templates/service.yaml

```diff
@@ -21,5 +21,11 @@ spec:
       {{- if and (eq (.Values.deploy.expose.type) "NodePort") (.Values.deploy.expose.nodePort) }}
       nodePort: {{ .Values.deploy.expose.nodePort }}
       {{- end}}
+    - port: 9999
```

Comment:
We should only expose this port if jmx is enabled and I don't think we want to expose it via a nodeport as the port value from the template `.Values.deploy.expose.nodePort` will already be used by the "infinispan" port defined above.

I think the code should be more like:

```yaml
   {{- if .Values.deploy.jmx.enabled }}
    - port: 9999
      protocol: TCP
      name: jmx
   {{- end}}
```

Reply:
Updated . Thanks